.github/CODEOWNERS

```
@@ -26,7 +26,7 @@
 # Todo: is this file still needed?
 /Makefile.trace                         @DataDog/agent-platform
 
-/release.json                           @DataDog/agent-platform @DataDog/agent-shared-components
+/release.json                           @DataDog/agent-platform @DataDog/agent-shared-components @DataDog/agent-metrics-logs
```

```suggestion
/release.json                           @DataDog/agent-platform @DataDog/agent-shared-components @DataDog/agent-metrics-logs @DataDog/windows-agent 
```
Similarly to the jmxfecth version, as you assess the NPM driver from release.json `windows-agent` could be put owner of the file as well, wdyt?

---

That makes sense in fact, it's just `windows-kernel-integrations` team in fact. I will add that part